src/js/components/Terminal/index.js

```diff
@@ -502,6 +502,9 @@ class Terminal extends Component {
       });
       e.target.value = ''; // eslint-disable-line no-param-reassign
     }
+    if (this.props.afterChangeHandler !== null) {
```

Comment:
If you were to change this if statement to a check for typeof 'function' that would be better